src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenSwitchedToSoleTest.java

```diff
@@ -352,6 +354,72 @@ void shouldNotUnlinkApp2OnAboutToSubmitCallback() {
         verify(ccdAccessService, times(0)).unlinkUserFromApplication(anyString(), anyLong(), anyString());
     }
 
+    @Test
+    void givenEventStartedWithValidJointCaseShouldRemoveJurisdictionAnswers() {
+        final long caseId = 1L;
+        CaseData caseData = validJointApplicant1CaseData();
+        setValidCaseInviteData(caseData);
+        caseData.setCaseInvite(new CaseInvite(TEST_APPLICANT_2_EMAIL, ACCESS_CODE, null));
+        caseData.setApplicant2(
+            Applicant.builder()
+                .firstName("Bob")
+                .middleName("The")
+                .lastName("Build")
+                .email("bob@buildings.com")
+                .gender(MALE)
+                .financialOrder(YES)
+                .lastNameChangedWhenMarried(YES)
+                .address(
+                    AddressGlobalUK.builder()
+                        .addressLine1("123 The Street")
+                        .postTown("The town")
+                        .county("County Durham")
+                        .country("England")
+                        .postCode("POSTCODE")
+                        .build())
+                .contactDetailsType(PRIVATE)
+                .build()
+        );
```

Comment:
I don't think this is necessary since we're only testing the jurisdiction fields here